lib/importer.py

```diff
@@ -236,7 +236,7 @@ def linkToMyPlexAccount() -> MyPlexAccount:
         return None
 
     # show the user the pin
-    dialog.ok(localize(32015), localize(32053), f"[COLOR FFE5A00D]{pinLogin.pin}[/COLOR]")
```

Comment:
Ah the problem is that some APIs have been changed in Kodi v19 nightlies. Before it was possible to pass multiple lines to `Dialog.ok()`. But obviously this information is still needed.